src/components/MultiHopTrade/components/TradeInput/components/MaxSlippage.tsx

```diff
@@ -0,0 +1,180 @@
+import { Divider, Flex, Skeleton, Stack } from '@chakra-ui/react'
+import { Tag } from '@chakra-ui/tag'
+import { bnOrZero } from '@shapeshiftoss/chain-adapters'
+import type { AmountDisplayMeta, SwapSource } from '@shapeshiftoss/swapper'
+import { useCallback, useMemo } from 'react'
+import { useTranslate } from 'react-polyglot'
+import { Amount } from 'components/Amount/Amount'
+import { Row } from 'components/Row/Row'
+import { Text } from 'components/Text'
+import type { TextPropTypes } from 'components/Text/Text'
+import { getChainAdapterManager } from 'context/PluginProvider/chainAdapterSingleton'
+import { useLocaleFormatter } from 'hooks/useLocaleFormatter/useLocaleFormatter'
+import { fromBaseUnit } from 'lib/math'
+import {
+  THORCHAIN_LONGTAIL_STREAMING_SWAP_SOURCE,
+  THORCHAIN_STREAM_SWAP_SOURCE,
+} from 'lib/swapper/swappers/ThorchainSwapper/constants'
+import { selectUserSlippagePercentage } from 'state/slices/tradeInputSlice/selectors'
+import { selectDefaultSlippagePercentage } from 'state/slices/tradeQuoteSlice/selectors'
+import {
+  convertDecimalPercentageToBasisPoints,
+  subtractBasisPointAmount,
+} from 'state/slices/tradeQuoteSlice/utils'
+import { useAppSelector } from 'state/store'
+
+type MaxSlippageProps = {
+  swapSource?: SwapSource
+  isLoading?: boolean
+  amountCryptoPrecision: string
+  slippageDecimalPercentage: string
+  hasIntermediaryTransactionOutputs?: boolean
+  intermediaryTransactionOutputs?: AmountDisplayMeta[]
+  symbol: string
+}
+
+export const MaxSlippage: React.FC<MaxSlippageProps> = ({
+  swapSource,
+  isLoading,
+  amountCryptoPrecision,
+  slippageDecimalPercentage,
+  hasIntermediaryTransactionOutputs,
+  intermediaryTransactionOutputs,
+  symbol,
+}) => {
+  const translate = useTranslate()
+  const {
+    number: { toCrypto },
+  } = useLocaleFormatter()
+  const shouldShowSlippageMinAmount = useMemo(
+    () =>
+      swapSource !== THORCHAIN_STREAM_SWAP_SOURCE &&
+      swapSource !== THORCHAIN_LONGTAIL_STREAMING_SWAP_SOURCE,
+    [swapSource],
+  )
+  const defaultSlippagePercentage = useAppSelector(selectDefaultSlippagePercentage)
+  const userSlippagePercentage = useAppSelector(selectUserSlippagePercentage)
+  const slippageAsPercentageString = bnOrZero(slippageDecimalPercentage).times(100).toString()
+  const isAmountPositive = bnOrZero(amountCryptoPrecision).gt(0)
+
+  const renderSlippageTag = useMemo(() => {
+    if (bnOrZero(defaultSlippagePercentage).eq(bnOrZero(slippageAsPercentageString))) {
+      return (
+        <Tag colorScheme='blue' size='sm'>
+          {translate('trade.slippage.auto')}
+        </Tag>
+      )
+    } else if (bnOrZero(slippageAsPercentageString).eq(bnOrZero(userSlippagePercentage))) {
```

Comment:
Can ditch the `else if` and have a next line `if` instead for indent cleanness

Reply:
no probs 8a0917ae8678cf65921bbef34766643c9ae05209